core/src/ons/platform.js

```diff
@@ -115,7 +115,13 @@ class Platform {
       (window.screen.width === 375 && window.screen.height === 812 || // X, XS portrait
        window.screen.width === 812 && window.screen.height === 375 || // X, XS landscape
        window.screen.width === 414 && window.screen.height === 896 || // XS Max, XR portrait
-       window.screen.width === 896 && window.screen.height === 414); // XS Max, XR landscape
+       window.screen.width === 896 && window.screen.height === 414 || // XS Max, XR landscape
+       window.screen.width === 360 && window.screen.height === 780 || // 12 Mini portrait
```

Comment:
Wrong values here but since we already check for the right values above, these can be deleted safely.